### DISCUSSION TITLE
My website hosted on Github Pages has stopped working since I uploaded it to Google Search Console

### DISCUSSION THREAD

**Discussion by EmanueleCiotola** [community/community#59451] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Guys, I really need some help. I designed a site to be shown at my State Exam and uploaded it to Google Search Console to make it findable. Since I've done this, every time I enter the site, the Javascript files on the site receive a 404 error. It's been 3 days since the upload to the google console and I need to fix this problem asap. The exam is near.

**Comment by yoannchaudet** [comment]
Would you mind posting a link to the site?

**Reply by EmanueleCiotola** [reply]
With the help of Google support I was able to fix the problem. It was their mistake, a bug. I just deleted the site from the Console and reloaded it
